MantidPlot/src/Mantid/MantidMatrixFunction.cpp

```diff
@@ -162,15 +161,19 @@ MantidMatrixFunction::getHistogramX(int row) const {
  * Y:       6      6       16        6         6
  * X: 2000    4000    8000    12000     16000     20000
  *
- * The algorithm will determine that the index of X which is closest to 6500 is 2,
- * but the Y index with the correct data is 1 (since the value should be 6 not 16)
+ * The algorithm will determine that the index of X which is closest to 6500 is
+ *2,
+ * but the Y index with the correct data is 1 (since the value should be 6 not
+ *16)
  *
  * 2. Point Data:
  * Y:   6      6       16        6         6
  * X: 2000    4000    8000    12000     16000
  *
- * The algorithm will determine that the index of X which is closest to 6500 is 2,
- * and the Y index with the correct data is 2 as well since there is a one-to-one
+ * The algorithm will determine that the index of X which is closest to 6500 is
+ *2,
+ * and the Y index with the correct data is 2 as well since there is a
+ *one-to-one
```

Comment:
Yes, that is consequence of clang formatting. I could try to work around that, but since this will need to be merged with your work, we can do this in one go.